### PR TITLE
DO NOT MERGE: playground only upgrade

### PR DESCRIPTION
**Pull request by danwt**
## Description

----

Closes #XXX

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow-up issues.

PR review checkboxes:

I have...

- [ ]  Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ]  Targeted PR against the correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to the GitHub issue with discussion and accepted design
- [ ]  Targets only one GitHub issue
- [ ]  Wrote unit and integration tests
- [ ]  Wrote relevant migration scripts if necessary
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)
- [ ]  Updated the scripts for local run, e.g genesis_config_commands.sh if the PR changes parameters
- [ ]  Add an issue in the [e2e-tests repo](https://github.com/dymensionxyz/e2e-tests) if necessary

SDK Checklist
- [ ] Import/Export Genesis
- [ ] Registered Invariants
- [ ] Registered Events
- [ ] Updated openapi.yaml
- [ ] No usage of go `map`
- [ ] No usage of `time.Now()`
- [ ] Used fixed point arithmetic and not float arithmetic
- [ ] Avoid panicking in Begin/End block as much as possible
- [ ] No unexpected math Overflow
- [ ] Used `sendCoin` and not `SendCoins`
- [ ] Out-of-block compute is bounded
- [ ] No serialized ID at the end of store keys
- [ ] UInt to byte conversion should use BigEndian

Full security checklist [here](https://www.faulttolerant.xyz/2024-01-16-cosmos-security-1/)

----

For Reviewer:

- [ ]  Confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  Confirmed all author checklist items have been addressed

----

After reviewer approval:

- [ ]  In case the PR targets the main branch, PR should not be squash merge in order to keep meaningful git history.
- [ ]  In case the PR targets a release branch, PR must be rebased.

### CODE REVIEW

File: app/upgrades/v5/v5.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+	_ upgrades.BaseAppParamManager,
+	keepers *keepers.AppKeepers,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		log := ctx.Logger().With("upgrade", UpgradeName)
+
+		p := keepers.RollappKeeper.GetParams(ctx)
```
Comment (omritoptix) [inline app/upgrades/v5/v5.go:51]: won't it fail here since old rollapps don't have the `MinSequencerBondGlobal` ?

File: app/upgrades/v5/v5.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		p.MinSequencerBondGlobal = rollapptypes.DefaultMinSequencerBondGlobalCoin
+		keepers.RollappKeeper.SetParams(ctx, p)
+		rollapps := keepers.RollappKeeper.GetAllRollapps(ctx)
+		for _, ra := range rollapps {
+			ra.MinSequencerBond = sdk.NewCoins(rollapptypes.DefaultMinSequencerBondGlobalCoin)
+			if ra.GenesisState.GetDeprecatedBridgeOpen() {
```
Comment (omritoptix) [inline app/upgrades/v5/v5.go:57]: this checks proof height but all of them will proof height = 0 before ugprade

File: app/upgrades/v5/upgrade.go [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+const (
+	UpgradeName = "v5"
+)
+
+var Upgrade = upgrades.Upgrade{
+	Name:          UpgradeName,
+	CreateHandler: CreateUpgradeHandler,
+	StoreUpgrades: storetypes.StoreUpgrades{
+		Added: []string{
+			consensustypes.ModuleName,
```
Comment (omritoptix) [inline app/upgrades/v5/upgrade.go:43]: don't think anything should be here as they were already added in v4 so they are not new.

Comment (danwt) [reply app/upgrades/v5/upgrade.go:43]: right good spot

**Review by keruch — DISMISSED** [review]

File: app/upgrades/v5/upgrade.go [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+					ra.GenesisState.TransferProofHeight = h
+				}
+			}
+			keepers.RollappKeeper.SetRollapp(ctx, ra)
+		}
+		return fromVM, nil
+	}
+}
+
+func setKeyTables(keepers *keepers.AppKeepers) {
```
Comment (keruch) [inline app/upgrades/v5/upgrade.go:74]: do you need it?

**Review by mtsitrin — APPROVED** [review]

**Pull request closed without merging**